sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala

```diff
@@ -25,6 +25,9 @@ import scalanative.sbtplugin.testinterface.ScalaNativeFramework
 
 object ScalaNativePluginInternal {
 
+  val optimizerReporterOutPath =
```

Comment:
Is it ok to have a key for the reporter back if it's in `ScalaNativePluginInternal`? @densh @Duhemm

Reply:
Does it make sense to have a build relative default directory?

Reply:
Well, we could have a switch to turn it on and off and have a project relative destination in the target directory. I think that would work for each project that uses the plugin. @Duhemm What do you think?